docroot/profiles/humsci/su_humsci_profile/su_humsci_profile.install

```diff
@@ -1026,3 +1028,113 @@ function su_humsci_profile_update_9714() {
     $role->save();
   }
 }
+
+/**
+ * Remove any existing hs_webform paragraph entities.
+ */
+function su_humsci_profile_update_9715($sandbox) {
+  $paragraph_storage = \Drupal::entityTypeManager()->getStorage('paragraph');
+  $paragraph_machine_name = 'hs_webform';
+
+  if (empty($sandbox['ids'])) {
+    $sandbox['ids'] = $paragraph_storage->getQuery()
+      ->condition('type', $paragraph_machine_name)
+      ->accessCheck(FALSE)
+      ->execute();
+    $sandbox['total'] = count($sandbox['ids']);
+  }
+  $paragraph_ids = array_splice($sandbox['ids'], 0, 50);
+
+  foreach ($paragraph_storage->loadMultiple($paragraph_ids) as $paragraph) {
+    $paragraph->delete();
+  }
+
+  $sandbox['#finished'] = count($sandbox['ids']) ? 1 - count($sandbox['ids']) / $sandbox['total'] : 1;
+}
+
+/**
+ * Delete hs_webform paragraph type and remove webform config.
+ */
+function su_humsci_profile_update_9716() {
+  $paragraph_machine_name = 'hs_webform';
+  // Delete the field and field storage on the paragraph.
+  $fields = \Drupal::entityTypeManager()
+    ->getStorage('field_config')
+    ->loadByProperties(['bundle' => $paragraph_machine_name]);
+
+  if ($fields) {
```

Comment:
You don't need to check for this. If it's an empty array, the foreach will just be skipped. 

But the fun fact is that you can delete the field storage and it'll automatically delete any field config associated to it.

Reply:
Thanks for the tip! I removed this section because we're removing the storage below.